src/bench/gcs_filter.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+#include <random.h>
+#include <blockfilter.h>
+
+static void ConstructGCSFilter(benchmark::State& state)
+{
+    std::set<GCSFilter::Element> elements;
+    for (int i = 0; i < 10000; i++) {
+        GCSFilter::Element element(32);
+        GetRandBytes(element.data(), element.size());
```

Comment:
Note use of `GetRandBytes` here as well. Potential inconsistency in test outcomes and potential slowdown.